addons/mail/controllers/discuss.py

```diff
@@ -149,7 +149,7 @@ def mail_channel_attachment(self, channel_id, attachment_id, **kwargs):
         channel_partner_sudo = request.env['mail.channel.partner']._get_as_sudo_from_request_or_raise(request=request, channel_id=int(channel_id))
         if not channel_partner_sudo.env['ir.attachment'].search([('id', '=', int(attachment_id)), ('res_id', '=', int(channel_id)), ('res_model', '=', 'mail.channel')], limit=1):
             raise NotFound()
-        return channel_partner_sudo.env['ir.http']._get_content_common(res_id=int(attachment_id), download=True)
+        return channel_partner_sudo.env['ir.http']._get_content_common(res_id=int(attachment_id))
```

Comment:
The download button still works without this parameter? Shouldn't `download` be a parameter of the route like in `/web/content`?